Demo/Demo/Models/ClientIDRequest.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import CorePayments
+
+public struct ClientIDRequest: APIRequest {
+
+    public typealias ResponseType = ClientIDResponse
+
+    public var path: String = "/client_id"
+    public var method: HTTPMethod = .get
+    public var body: Data?
+
+    public var headers: [HTTPHeader: String] {
+        [
+            .accept: "application/json",
+            .acceptLanguage: "en_US"
+        ]
+    }
+}
```

Comment:
If this is all just for the demo app we probably can have it be internal vs public

Reply:
Addressed in commit 8e0b7395b40d47023968064c74cd7ea89e7b78ec